tables/person_comps_extra.sql

```diff
@@ -1,15 +1,15 @@
-DROP TABLE IF EXISTS personCompsExtra;
+DROP TABLE IF EXISTS person_Comps_Extra;
 SET @p = NULL, @c = 0, @dd = NULL, @d = NULL;
-CREATE TABLE personCompsExtra
+CREATE TABLE person_Comps_Extra
 SELECT a.*,
 	@c := IF(a.personId = @p,@c+1,1) 'compNumber',
 	@dd := IF(a.personId = @p,DATEDIFF(a.date,@d),NULL) 'daysLastComp',
 	@p := a.personId 'drop1',
 	@d := a.date 'drop2'
 FROM
-	(SELECT a.*, b.PBs
+	(SELECT a.*, b.PBs, a.singlePBs, a.averagePBs
```

Comment:
These should be `b.` not `a.`... right?

Reply:
yup I literally just realised that with my own code and came here to change it